sufia-models/app/models/concerns/sufia/works/generic_work.rb

```diff
@@ -5,7 +5,9 @@ module GenericWork
 
     include Sufia::Works::Work
     include Sufia::Works::CurationConcern::WithBasicMetadata
-    include Sufia::GenericFile::Permissions
+    include Sufia::GenericFile::Batches
```

Comment:
@tingtingjh As discussed on the call.  If you could just add a TODO to remove these times once the collection size and processing tickets are closed. #1120 & #1122 .  Thanks!